libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -64,6 +61,16 @@ class AP_RCProtocol {
 #endif
         NONE    //last enum always is None
     };
+
+    // return protocol name as a string
+    static const char *protocol_name_from_protocol(rcprotocol_t protocol);
+
+#if AP_RCPROTOCOL_ENABLED
```

Comment:
We still define AP_RCProtocol the class because we need to be able to get the strings and backend-type-enumeration in the case that you have an IOMCU but no AP_RCProtocol built in.